### ISSUE TITLE
Unable to add Lambda Trigger to graphQL @model DynamoDB tables

### ISSUE THREAD

**Issue by jon144** [aws-amplify/amplify-cli#4042]
**Describe the bug**
When adding a Lambda trigger to a DynamoDB @model table the following error is thrown.  Error: Unknown template type undefined.

**Amplify CLI Version**
4.18.0

**To Reproduce**
Setup:
Create new amplify project.
Add GraphQL API with API key (for simplicity)
Use the Todo single table schema.
Push API to cloud. 
At this point, everything is working as expected.

Create the error:
PS C:\source\repos\demo1> amplify function add
Using service: Lambda, provided by: awscloudformation
? Provide a friendly name for your resource to be used as a label for this category in the project: functionJ
? Provide the AWS Lambda function name: functionJ
? Choose the function runtime that you want to use: **.NET Core 3.1**
? Choose the function template that you want to use: Trigger (DynamoDb, Kinesis)
? What event source do you want to associate with Lambda trigger? Amazon DynamoDB Stream  
? Choose a DynamoDB event source option Use API category graphql @model backed DynamoDB table(s) in the current Amplify project
Selected resource demo1
Selected @model Todo
Error: Unknown template type undefined
    at Object.provideTrigger (C:\Users\XXX\AppData\Roaming\npm\node_modules\@aws-amplify\cli\node_modules\amplify-dotnet-function-template-provider\lib\providers\triggerProvider.js:24:19)
    at async Object.templateWalkthrough (C:\Users\XXX\AppData\Roaming\npm\node_modules\@aws-amplify\cli\node_modules\amplify-category-function\lib\provider-utils\awscloudformation\utils\functionPluginLoader.js:37:12)
    at async Object.createWalkthrough (C:\Users\XXX\AppData\Roaming\npm\node_modules\@aws-amplify\cli\node_modules\amplify-category-function\lib\provider-utils\awscloudformation\service-walkthroughs\lambda-walkthrough.js:23:70)
    at async Object.addResource (C:\Users\XXX\AppData\Roaming\npm\node_modules\@aws-amplify\cli\node_modules\amplify-category-function\lib\provider-utils\awscloudformation\index.js:29:9)
    at async Object.executeAmplifyCommand (C:\Users\XXX\AppData\Roaming\npm\node_modules\@aws-amplify\cli\node_modules\amplify-category-function\lib\index.js:130:5)
    at async executePluginModuleCommand (C:\Users\JonGross\AppData\Roaming\npm\node_modules\@aws-amplify\cli\lib\execution-manager.js:131:13)
    at async Object.executeCommand (C:\Users\XXX\AppData\Roaming\npm\node_modules\@aws-amplify\cli\lib\execution-manager.js:16:9)
    at async Object.run (C:\Users\XXX\AppData\Roaming\npm\node_modules\@aws-amplify\cli\lib\index.js:45:9)
There was an error adding the function resource

**Expected behavior**
Create a lambda trigger function connected to the Todo table

**Screenshots**
none

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Node: v12.16.2 

**Additional context**
none

**Comment by andyhopp** [comment]
Hello! The code change that resolves this issue will be in the next release, but in the meantime you can clone/build a local copy of amplify CLI that can run side-by-side with the regular release. (see: https://github.com/aws-amplify/amplify-cli#contributing on how to build/use amplify-dev)

**Comment by edwardfoyle** [comment]
The fix for this has been released in Amplify v 4.18.1

**Comment by jon144** [comment]
I have installed Amplify 4.18.1 and still get the error. 

To upgrade amplify I ran
`npm install -g @aws-amplify/cli`

To check the version I ran
`amplify -version`
and 4.18.1 was returned.

Here are my steps.
```
PS C:\Users\XXX\source\repos\awszzz> amplify add function
Using service: Lambda, provided by: awscloudformation
? Provide a friendly name for your resource to be used as a label for this category in the project: triggerConnection
? Provide the AWS Lambda function name: triggerConnection
? Choose the function runtime that you want to use: .NET Core 3.1
? Choose the function template that you want to use: Trigger (DynamoDb, Kinesis)
? What event source do you want to associate with Lambda trigger? Amazon DynamoDB Stream  
? Choose a DynamoDB event source option Use API category graphql @model backed DynamoDB table(s) in the current Amplify project
Selected resource awszzz
? Choose the graphql @model(s) Connection
Error: Unknown template type undefined
    at Object.provideTrigger (C:\Users\XXX\AppData\Roaming\npm\node_modules\@aws-amplify\cli\node_modules\amplify-dotnet-function-template-provider\lib\providers\triggerProvider.js:24:19)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Object.templateWalkthrough (C:\Users\XXX\AppData\Roaming\npm\node_modules\@aws-amplify\cli\node_modules\amplify-category-function\lib\provider-utils\awscloudformation\utils\functionPluginLoader.js:37:12)
    at async Object.createWalkthrough (C:\Users\XXX\AppData\Roaming\npm\node_modules\@aws-amplify\cli\node_modules\amplify-category-function\lib\provider-utils\awscloudformation\service-walkthroughs\lambda-walkthrough.js:25:70)
    at async Object.addResource (C:\Users\XXX\AppData\Roaming\npm\node_modules\@aws-amplify\cli\node_modules\amplify-category-function\lib\provider-utils\awscloudformation\index.js:29:9)
    at async Object.executeAmplifyCommand (C:\Users\XXX\AppData\Roaming\npm\node_modules\@aws-amplify\cli\node_modules\amplify-category-function\lib\index.js:130:5)
    at async executePluginModuleCommand (C:\Users\XXX\AppData\Roaming\npm\node_modules\@aws-amplify\cli\lib\execution-manager.js:131:13)
    at async Object.executeCommand (C:\Users\XXX\AppData\Roaming\npm\node_modules\@aws-amplify\cli\lib\execution-manager.js:16:9)
    at async Object.run (C:\Users\XXX\AppData\Roaming\npm\node_modules\@aws-amplify\cli\lib\index.js:45:9)
There was an error adding the function resource

```

Is there anything else I need to do?

**Comment by edwardfoyle** [comment]
@jon144 Sorry for the confusion, I thought the fix for this had gone out, but it is still pending release. It'll be available in the NEXT release of the CLI. Sorry about that!

**Comment by edwardfoyle** [comment]
@jon144 This fix is now released in v 4.19.0
